### PR TITLE
APP-1208 Initial support for app authentication

### PR DESCRIPTION
**Pull request by robson-sanchez**

### CODE REVIEW

File: services/connectService.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+/* eslint-disable no-unused-vars */
+import React from 'react';
+import { render } from 'react-dom';
+import { Provider } from 'react-redux';
+import factory from '../js/config.service';
+import configureStore from '../store/configureStore';
+import '../vendors/font-awesome-4.6.3/css/font-awesome.min.css';
```
Comment (luanapp) [inline services/connectService.js]: Is this css related with the connection service?

File: services/registerService.js [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+* @return       SYMPHONY.remote.hello       returns a SYMPHONY remote hello service.
+*/
+export const register = (SYMPHONY, config, enrichers) => {
+  const controllerName = `${config.appId}:controller`;
+
+  let exportedDependencies = enrichers ? enrichers.map((enricher) => enricher.name) : [];
+  exportedDependencies.push(controllerName);
+
+  // create our own service
+  const listService = SYMPHONY.services.register(controllerName);
```
Comment (luanapp) [inline services/registerService.js]: this variable can have a clearer name

**Review by luanapp — APPROVED** [review]

File: services/connectService.js (hunk 1 above)
Comment (robson-sanchez) [reply services/connectService.js]: No, it isn't. I'm gonna remove it.

File: services/registerService.js (hunk 2 above)
Comment (robson-sanchez) [reply services/registerService.js]: DONE
